.gitlab/olcf-gitlab-ci.yml

```diff
@@ -8,16 +8,10 @@ hipcc:
     - module load rocm/6.0
     - export CMAKE_BUILD_PARALLEL_LEVEL=48
     - export ENV_CMAKE_OPTIONS=""
-    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-D CMAKE_CXX_COMPILER=hipcc"
-    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-D Kokkos_ENABLE_HIP=ON"
-    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-D Kokkos_ENABLE_TESTS=ON"
-    - ctest -VV
-        -E Kokkos_CoreUnitTest_DeviceAndThreads
-        -D CDASH_MODEL="Nightly"
-        -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}"
-        -S CTestRun.cmake
-        -D CTEST_SITE="frontier"
-        -D CTEST_BUILD_NAME="hipcc-rocm/6.0"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DCMAKE_CXX_COMPILER=hipcc"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_HIP=ON"
+    - export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_TESTS=ON"
```

Comment:
What changed here?  The space between -D and the argument?  Is it needed or a matter of taste?
(I don't care too much, just trying to understand what is going on)

Reply:
We cannot have a blank space between -D and the argument here but we need to have one between -D and the argument in the `ctest` line.